hr_expense_sequence/__openerp__.py

```diff
@@ -27,6 +27,7 @@
     'author': "Serv. Tecnol. Avanzados - Pedro M. Baeza,"
               "Odoo Community Association (OCA)",
     'website': 'http://www.serviciosbaeza.com',
```

Comment:
URL should be `https://github.com/OCA/hr`

Reply:
This module is not related with this PR

Reply:
> This module is not related with this PR

Ok, Noted!!